projects/assets-library/assets/styles/_font.scss

```diff
@@ -92,6 +92,7 @@ $font-family: 'Work Sans', sans-serif;
 @mixin ellipsis-overflow {
   white-space: nowrap;
   overflow: hidden;
+  overflow: -webkit-paged-x; // For safari
```

Comment:
Can we fix these without browser specifics? Most of these issues are things like safari requires a width when other browsers don't etc.

Reply:
will check. I'm also looking at other issues on safari. will see if I can replace this.